src/core.ts

```diff
@@ -1145,7 +1145,25 @@ function applyHeadersMut(targetHeaders: Headers, newHeaders: Headers): void {
 }
 
 export function debug(action: string, ...args: any[]) {
+  const sensitiveHeaders = ["authorization", "Authorization", "api-key"];
   if (typeof process !== 'undefined' && process?.env?.['DEBUG'] === 'true') {
+    for (const arg of args) {
+      if (!arg) {
+        break;
+      }
+      if (arg["headers"]) {
+        for (const header of sensitiveHeaders) {
```

Comment:
Can we make the check case-insensitive?
```suggestion
  const sensitiveHeaders = ["authorization", "api-key"];
  if (typeof process !== 'undefined' && process?.env?.['DEBUG'] === 'true') {
    for (const arg of args) {
      if (!arg) {
        break;
      }
      if (arg["headers"]) {
        for (const header.toLowerCase() of sensitiveHeaders) {
```